src/Gui/DlgSettingsWorkbenchesImp.cpp

```diff
@@ -212,8 +212,6 @@ void wbListItem::onWbToggled(bool checked)
     loadLabel->setEnabled(checked);
     loadButton->setEnabled(checked);
     autoloadCheckBox->setEnabled(checked);
```

Comment:
I prefer not because I think this usecase is very very usecase, and what drmacro described feels very much like a hack to get a missing feature in core.

I would let it disabled because it visually shows that the workbench is disabled.